templates/telegraf.conf.j2

```diff
@@ -204,3 +204,18 @@
 ###############################################################################
 #                              service PLUGINS                                #
 ###############################################################################
+
+###############################################################################
+#                                  PROCESSORS                                    #
+###############################################################################
+
+{% if telegraf_processors is defined and telegraf_processors is iterable %}
+{% for item in telegraf_processors %}
+[[processors.{{ item.processor }}]]
+{% if item.config is defined and item.config is iterable %}
+{% for items in item.config %}
```

Comment:
`items` isnt a great iterator variable =). How about `for itemConfig in item.configs`? and rename it to `configs`?

Reply:
so, its really 1 config per processor, so configs will confusing. Item here is a processor_plugin, and i will just call it that.

Reply:
i will change `item -> processor_plugin ` and
`items`-> `configs`